### ISSUE TITLE
TWIN: Edit Header & Footer

### ISSUE THREAD

**Issue by samtaggart** [threefoldfoundation/www_threefold_twin#57]
Header:
- [x] remove team (already done)
- [x] remove tf home (already done)
- [x] change knowledge base >>> learn
- [x] add 'more'
  - dropdown
     - Community >> create telegram for Dt & links to it: https://t.me/joinchat/JnJfqY9tfAU1NTY0
     - Blog >> create new blog for Dt (@Vilnite) 
     - Forum >> create new forum for Dt (@Vilnite)
     - Contact Us >> create email for Dt -  info@mydigitaltwin.com (@sabrinasadik / @VonSub) 
     

- [x] Change order in dropdown of experiences to:
   - Chat
   - Meet
   - File Storage
   - Office
   - Web Publisher
   - Payments 

- [x] Footer -> see info below

*Digital Twin*
- About Digital Twin >>> https://mydigitaltwin.io/info/twin#/
- Our Vision >> https://mydigitaltwin.io/info/twin#/twin__our_vision
- Why Peer-to-Peer >> https://mydigitaltwin.io/info/twin#/twin__power_of_p2p
- Your Unique Digital ID >>> https://mydigitaltwin.io/info/twin#/twin__one_digital_identity

*Experiences* >> lead to website pages
- Chat
- Meet
- File Storage
- Office
- Web Publisher
- Payments 

*Get Started*
- Get Your Digital Twin >>> https://mydigitaltwin.io/info/twin#/twin__getting_started
- Pricing >>> https://mydigitaltwin.io/info/twin#/twin__price
- Get TFT >> https://info.mydigitaltwin.io/#/twin__how_to_get_tft
- Store TFT >> https://info.mydigitaltwin.io/#/twin__how_to_store_tft

*Community*
- Join Community https://t.me/joinchat/JnJfqY9tfAU1NTY0
- Blog 
- Forum
- Contact Us

*Bottom of footer* (same as threefold.io >> but leads to documents unique to DT)
- Terms & Conditions
- Privacy policy
- Impressum

**Comment by samtaggart** [comment]
Talked to @Sacha96 we need to do this before Thursday.

**Comment by Sacha96** [comment]
@Vilnite created way forward with @AlexandreHannelas. This is not prior for the week but needs to be done before Sunday. I let you organize according to workload. thanks.

Though if possible @AlexandreHannelas try assist in creating necessary links so we can have it done by Thursday. i let you two coordinate.

**Comment by Vilnite** [comment]
For DT Forum created an issue https://github.com/threefoldfoundation/www_threefold_twin/issues/68
and asked Roel's help there.
Will update Forum links on DT website when it's created.
